codes/depression_among_adults.qmd

```diff
@@ -7,119 +7,114 @@ date: "2025-02-15"
-  labs(title = "Trends in Adult Depression (2012-2018)",
-       x = "Year", y = "Prevalence (%)")
+## or we can also say The column `r colnames(depression_data)[which(colnames(depression_data) == "Year)]`
 
-```
+- `r colnames(depression_data)[2]` represents a category used to divide the data (eg race, age,income). 
+- `r colnames(depression_data)[3]` specifies the subgroup within the Strata.  
+- `r colnames(depression_data)[4]` represents the count of individuals in the given category.  
+- `r colnames (depression_data)[5]`  provides a weighted estimate of the number of people in the population.  
+- `r colnames(depression_data)[6] ` represents the percentage of the population experiencing depression condition.  
+- `r colnames(depression_data)[7]` represents the lower bound of the 95% confidence interval.  
+- `r colnames(depression_data)[8]` represents the upper bound of the 95% confidence interval.
 
-
-- are there  significant difference in depression rates between males and females over the years?
+## Data analysis 
 
 ```{r}
-
-
-sex_depression <- depression_data %>%
-  filter(Strata == "Sex") %>%
-  ggplot(aes(x = Strata.Name, y = Percent, fill = Strata.Name)) +
-  geom_bar(stat = "identity", position = "dodge") +
-  labs(title = "Depression Rates in Males and Females Over the Years",
-       x = "Sex", y = "Prevalence (%)") +
-  theme_minimal()
-
-
+# filter out rows with Sex and Total in the 'Strata' column
+data_1 <- depression_data %>%
+  dplyr::filter(!(Strata %in% c("Total", "Sex")))
+
+# plot the depression level distribution across the different strata over years
+p <- ggplot(data_1, aes(fill = Strata.Name, x = Strata, y = Percent)) +
+  geom_bar(position = "dodge", stat = "identity") +
+  facet_wrap(~Year)
 ```
 
-
-- How does race/ethnicity influence depression rates?
 ```{r}
-ggplot(depression_data, aes(x = Strata.Name, y = Percent, fill = Strata.Name)) +
-  geom_bar(stat = "identity") +
-  labs(title = "Depression Rates by Race/Ethnicity",
-       x = "Race/Ethnicity", y = "Prevalence (%)")
-
+ggplot(depression_data,
+       aes(x = Year, y = Percent, group = Strata.Name,
+           color = Strata.Name)) +
+  geom_line() + 
+  facet_wrap(~Strata)
 ```
 
 
-- Which ethnic group has the highest prevalence of depression over the years?
-
-- Is there a significant difference in depression rates based on income levels
-Statistical Associations
+- are there  significant difference in depression rates between males and females over the years?
 
-- Is there a correlation between weighted frequency and percentage of depression?
+## statistical tests
 
-- Are there significant differences in depression rates between different age groups ?
-
-- Show the distribution of depression rates across different groups.
+To know if there is a statistical difference in depression btw male and female we first need to check if the distribution is normal then we hypothesised that there is a statistical difference between two sexes. 
```

Comment:
This is not valid for when you are comparing proportions. 
Check this link https://www.sthda.com/english/wiki/two-proportions-z-test-in-r